lib/advanced-open-file-view.coffee

```diff
@@ -270,29 +272,38 @@ class AdvancedFileView extends View
 
     # Handle keyboard movement
     @miniEditor.on 'keydown', (ev) =>
-      if ev.keyCode is 9
+      if ev.keyCode is 9 # Tab
         blockEvent ev
         return
 
+      if ev.keyCode is 90 and (ev.ctrlKey or ev.metaKey) # Ctrl/Cmd + Z
```

Comment:
So, this makes ctrl-z work on Mac, which is a little weird, but probably hard to fix…